.github/workflows/unitcoverage.yaml

```diff
@@ -0,0 +1,24 @@
+name: Unit Test Coverage
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  test:
+    name: Unit Tests
+    runs-on: ubuntu-20.04
```

Comment:
Do they have a way to run unit tests on multiple OS's during a run?

Reply:
Yes, a matrix can be implemented, but the OS supported list is small. See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.
We can only run ubuntu, macos, and windows server.